### DISCUSSION TITLE
Random Fly

### DISCUSSION THREAD

**Discussion by Soleil-des-chats** [EssentialsX/Essentials#5239] [Q&A] [answered]
I own a server and my players can at random times fly on the server.
This plugin really suck because they don't have the permisions.
I already had problem with the bad plugin. Fix it or help me or i replace it with an better.

**Comment by Lunna5** [comment, marked as answer]
It can be that your players have this permission essentials.fly.safelogin try setting this permission to false on your permission plugin.

Then related to your ways to ask for help I don't think that is a good way

**Comment by SkyKingPX** [comment]
In the config.yml there is something like "enable-fly-on-join", which is set to true as default. Set it to false

Edit: I found it: send-fly-enable-on-join on line 645

**Reply by mdcfe** [reply]
That config option only hides the message when someone logs in. It does not change anything else that EssentialsX does when someone logs in.

**Reply by SkyKingPX** [reply]
No, on my Server it works now
